exporter/cat1_view_helper.go

```diff
@@ -241,3 +241,14 @@ func isNil(i interface{}) bool {
 func derefBool(i *bool) bool {
 	return *i
 }
+
+// RecordTarget is used for rendering the record_target template
+type RecordTarget struct {
```

Comment:
Why not just put the CMSCompatibility field on `models.Record`? That way the field would be set before the `cat1data` struct is made in `exporter.go` and then you would be able to feed the `models.Record` struct straight to the `_record_target.xml` template. Then this type and the code to create it could be avoided.